### PR TITLE
Update lessons on Git

### PR DESCRIPTION
**Pull request by encukou**
* Update examples of Git output (which are slightly different in recent versions)
* Show a merge conflict and its resolution
* Explain how provided names are used, and that we don't need real names
* Use [naucse/prezencka](https://github.com/naucse/prezencka) rather than my [personal repo](https://github.com/encukou/prezencka). (So far it's not too much load for me to merge the PRs, but let me know if you want access.)

### CODE REVIEW

File: lessons/git/collaboration/index.md [hunk 1]
```diff
@@ -154,26 +155,55 @@ a zkus se podívat na historii (`gitk` nebo `git log`).
...
 
-Spousta míst na Internetu funguje tak, že vybraná
-skupina lidí má „přístup”: můžou dělat změny,
-jak se jim líbí.
+Spousta míst na Internetu (blogy, zpravodajství, e-shopy) funguje tak, že
+vybraná skupina lidí, „editorů“, má právo měnit obsah, jak se jim líbí.
+Takovým editorům musí správce projektu věřit, než jim přístup povolí.
 
-S Gitem se používá jiný přístup:
+S Gitem se používá trožku jiný mechanismus:
```
Comment (adelpopelkova) [inline lessons/git/collaboration/index.md:203]:
```suggestion
S Gitem se používá trošku jiný mechanismus:
```

File: lessons/git/collaboration/index.md [hunk 2]
```diff
@@ -154,26 +155,55 @@ a zkus se podívat na historii (`gitk` nebo `git log`).
...
 změny nahraješ do *vlastního* sdíleného
-repozitáře a majiteli původního projektu napíšeš
+repozitáře, který máš právo měnit jen ty.
+Majiteli původního projektu pak napíšeš
 žádost o začlenění těch změn (angl. *pull request*).
 Může to být třeba mail se slovy „Hele, na té a té
 adrese mám nějaké změny, které by se ti mohli hodit!
```
Comment (adelpopelkova) [inline lessons/git/collaboration/index.md:209]:
```suggestion
adrese mám nějaké změny, které by se ti mohly hodit!
```

File: lessons/git/collaboration/index.md [hunk 3]
```diff
@@ -204,12 +234,20 @@ adresa by měla být něco jako
...
-> máš u sebe na počítači.
-
+> být užitečné pro ostatní).
+> Tyhle dvě  kopie existují na serverech GitHubu a jsou volně dostupné
+> přes internet.
+>
+> Třetí kopii repozitáře pak máš u sebe na počítači.
+> K té se dostaneš jen ty.
+>
+> Z „hlavní“ verze si stáhnout práci ostatních členů týmu;
```
Comment (adelpopelkova) [inline lessons/git/collaboration/index.md:247]:
```suggestion
> Z „hlavní“ verze si stáhneš práci ostatních členů týmu;
```

Comment (adelpopelkova) [reply lessons/git/collaboration/index.md:247]: Or a different form of the word `stáhnout`

**Comment by encukou** [comment]
Thanks! Updated.

They output can't be changed fully automatically: the "create_examples.sh" script can generate the text, but it needs to be pasted into the lesson manually.

**Pull request merged**